app/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+from .models import User
+from django.contrib.auth.forms import UserCreationForm
```

Comment:
Thứ tự import các module là:
python package
django package
custom package

Nên đoạn này nên sửa thành 
```suggestion
from django import forms
from django.contrib.auth.forms import UserCreationForm

from .models import User
```

Reply:
oki